### PR TITLE
test on win32 py3.9 with pycurl windows wheels from https://www.lfd.uci.edu/~gohlke/pythonlibs/

### PR DESCRIPTION
**Pull request by graingert**
you're not supposed to use the wheels directly so I made my own mirror
on github pages
If you merge this I'll need you to move the repo into the celery org https://github.com/graingert/celery-wheelhouse/

*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -45,6 +45,7 @@ commands =
     unit: pytest --maxfail=10 -v --cov=celery --cov-report=xml --cov-report term {posargs}
     integration: pytest -xsv t/integration {posargs}
 setenv =
+    PIP_FIND_LINKS=https://graingert.co.uk/celery-wheelhouse/repo/simple/pycurl/
```
Comment (graingert) [inline tox.ini:48]: this didn't work
```suggestion
```

**Comment by auvipy** [comment]
why win 32?

**Comment by graingert** [comment]
> why win 32?

that's what `sys.platform` calls windows

**Comment by graingert** [comment]
@auvipy do note that I'll want to transfer in celery-wheelhouse to https://github.com/celery/celery-wheelhouse

File: tox.ini [hunk 2]
```diff
@@ -45,6 +45,7 @@ commands =
     unit: pytest --maxfail=10 -v --cov=celery --cov-report=xml --cov-report term {posargs}
     integration: pytest -xsv t/integration {posargs}
 setenv =
+    PIP_EXTRA_INDEX_URL=https://graingert.co.uk/celery-wheelhouse/repo/simple/
```
Comment (graingert) [inline tox.ini:48]: also happy to take other names on this. "hamster-wheels" ?

Comment (thedrow) [reply tox.ini:48]: I guess being explicit is better so I like the original name.

Comment (graingert) [reply tox.ini:48]:
```suggestion
    PIP_EXTRA_INDEX_URL=https://celery.github.io/celery-wheelhouse/repo/simple/
```

**Review by thedrow — APPROVED** [review]

**Review by thedrow — APPROVED** [review]

**Pull request merged**